cmake/kokkos_configure_trilinos.cmake

```diff
@@ -0,0 +1,35 @@
+IF(CMAKE_PROJECT_NAME STREQUAL "Trilinos")
+  set(Kokkos_ENABLE_SERIAL ON CACHE BOOL "Whether to build Serial backend" FORCE)
+
+  if(NOT ${Trilinos_ENABLE_OpenMP} STREQUAL "")
+    set(Kokkos_ENABLE_OPENMP ${Trilinos_ENABLE_OpenMP} CACHE BOOL "Whether to build OpenMP backend" FORCE)
+  else()
+    set(Kokkos_ENABLE_OPENMP                       OFF CACHE BOOL "Whether to build OpenMP backend" FORCE)
+  endif()
```

Comment:
Wonder if this repeated code block could be done with a single macro.

Reply:
Not sure if the added complexity is worth it for these few uses.